CamMerrill/Shackelford
# [WIP] BP_Brush

## Summary
Created new blueprint object `BP_Brush` under `Blueprints/GrabbableObjects/` that functions similarly to the trowel but destroys less of the voxel mesh than the trowel.

## Testing
1. Load Lab7 map
1. Place a `BP_Brush` objects in the scene and grab it
1. Teleport into the pit and try digging with the trowel and the brush to see the difference

## Notes
- Voxel removal is somewhat unrealistic with physical expectations of brush behavior - the collision handling probably needs to be changed in order to make it feel more like a brush
